docs/ops/movement/Pad_1.md

```diff
@@ -57,40 +27,71 @@ depending on the *pad_mode*.
 ```
 OUTPUT =
 [[ 0  1  2  3  4  0  0  0 ]
-[  0  5  6  7  8  0  0  0 ]
-[  0  9 10 11 12  0  0  0 ]
-[  0  0  0  0  0  0  0  0 ]
-[  0  0  0  0  0  0  0  0 ]]
+ [ 0  5  6  7  8  0  0  0 ]
+ [ 0  9 10 11 12  0  0  0 ]
+ [ 0  0  0  0  0  0  0  0 ]
+ [ 0  0  0  0  0  0  0  0 ]]
 ```
 * `pad_mode = "edge"`:
 ```
 OUTPUT =
 [[ 1  1  2  3  4  4  4  4 ]
-[  5  5  6  7  8  8  8  8 ]
-[  9  9 10 11 12 12 12 12 ]
-[  9  9 10 11 12 12 12 12 ]
-[  9  9 10 11 12 12 12 12 ]]
+ [ 5  5  6  7  8  8  8  8 ]
+ [ 9  9 10 11 12 12 12 12 ]
+ [ 9  9 10 11 12 12 12 12 ]
+ [ 9  9 10 11 12 12 12 12 ]]
 ```
 * `pad_mode = "reflect"`:
 ```
 OUTPUT =
-[[ 2  1  2  3  4  3  2  1 ]
-[  6  5  6  7  8  7  6  5 ]
-[ 10  9 10 11 12 11 10  9 ]
-[  6  5  6  7  8  7  6  5 ]
-[  2  1  2  3  4  3  2  1 ]]
+[[  2  1  2  3  4  3  2  1 ]
+ [  6  5  6  7  8  7  6  5 ]
+ [ 10  9 10 11 12 11 10  9 ]
+ [  6  5  6  7  8  7  6  5 ]
+ [  2  1  2  3  4  3  2  1 ]]
 ```
 * `pad_mode = "symmetric"`:
 ```
 OUTPUT =
 [[ 1  1  2  3  4  4  3  2 ]
-[  5  5  6  7  8  8  7  6 ]
-[  9  9 10 11 12 12 11 10 ]
-[  9  9 10 11 12 12 11 10 ]
-[  5  5  6  7  8  8  7  6 ]]
+ [ 5  5  6  7  8  8  7  6 ]
+ [ 9  9 10 11 12 12 11 10 ]
+ [ 9  9 10 11 12 12 11 10 ]
+ [ 5  5  6  7  8  8  7  6 ]]
 ```
 
-**Example**
+**Attributes**
+
+* *pad_mode*
+
+  * **Description**: *pad_mode* specifies the method used to generate new element values.
+  * **Range of values**: Name of the method in string format:
+    * `constant` - padded values are equal to the value of the *pad_value* input, if input not provided zero value is padded.
+    * `edge` - padded values are copied from the respective edge of the input `data` tensor.
+    * `reflect` - padded values are a reflection of the input `data` tensor; values on the edges are not duplicated. `pads_begin[D]` and `pads_end[D]` must be not greater than `data.shape[D] – 1` for any valid `D`.
+    * `symmetric` - padded values are symmetrically added from the input `data` tensor. This method is similar to the `reflect`, but values on edges are duplicated. Refer to the examples above for more details. `pads_begin[D]` and `pads_end[D]` must be not greater than `data.shape[D]` for any valid `D`.
+  * **Type**: string
+  * **Default value**: None
+  * **Required**: *yes*
+
+**Inputs**
+
+* **1**: `data` - input tensor to be padded. Required.
+
+* **2**: `pads_begin` - specifies the number of padding elements at the beginning of each axis. A list of non-negative integers. The length of the list must be equal to the number of dimensions in the input tensor. Required.
+
+* **3**: `pads_end` - specifies the number of padding elements at the beginning of each axis. A list of non-negative integers. The length of the list must be equal to the number of dimensions in the input tensor. Required.
```

Comment:
Missing information about shape. To describe type please use the pattern with _T_ and _Types_ section as in other specs.

Reply:
done